code/game/gamemodes/clock_cult/clock_ratvar.dm

```diff
@@ -10,6 +10,7 @@
 	poi_list += src
 
 /obj/structure/clockwork/massive/Destroy()
+	world << 'sound/magic/clockwork/anima_fragment_death.ogg'
```

Comment:
You should probably move this to `massive/ratvar` - gateways are also counted as massive structures.